src/test/loader.ts

```diff
@@ -132,7 +132,12 @@ export class Loader {
   }
 
   async loadReporter(file: string): Promise<new (arg?: any) => Reporter> {
-    let func = await this._requireOrImport(path.resolve(this._fullConfig.rootDir, file));
+    let func = '';
+    try {
+      func = await this._requireOrImport(path.resolve(this._fullConfig.rootDir, file));
```

Comment:
I think we need to be relative to the process.cwd(), not the rootDir. The config shouldn't change how my command line arguments are resolved.